### ISSUE TITLE
Notes on bass clef staff are not aligned with notes on treble clef staff. Offset by 1.5 px.

### ISSUE THREAD

**Issue by ronyeh** [0xfe/vexflow#601]
See: https://jsfiddle.net/Le5drt9k/6/

On my monitor, the time signature 4/4 is also not aligned. The one in the bass clef is offset to the right by about 1.5 pixels. 

The two middle C notes are not aligned. The stem in the bass clef is offset to the right by about 1.5 pixels. Look for the line that says `<== Try 38.5 instead!`

Is this because the bass clef symbol is wider than the treble clef symbol?

**Comment by ronyeh** [comment]
Thanks for your response! I didn't know about the voice API.

However, my feedback:

It's not intuitive to the user of your API that the same exact line of code will have different results depending on whether the note is added to the treble stave or bass stave:

```
tickContext.addTickable(note1);
tickContext.addTickable(note2);
tickContext.preFormat().setX(40); // This should set the note's x coordinate to 40
note1.draw();
note2.draw();
```

In the above example, if note1 is on the treble stave and note2 is on the bass stave, they will not be aligned, even though I've set their x coordinate to 40. It *feels* buggy because they are almost aligned and are 1.5 pixels off for a reason unknown to the user of the API.

Thanks,
Ron

**Comment by 0xfe** [comment]
Yeah, I hear ya, but `setX` is relative to the stave modifiers (like clef, timesig, etc.). It's really for the internal or custom formatters to use, not the end user. (I totally see how it feels buggy to use it.)

**Comment by ronyeh** [comment]
Ah. I didn't know it's an internal API. :-)

I was learning vexflow by looking at your tests page and then examining the associated JS file.

http://www.vexflow.com/build/vexflow-tests.js

The above code makes use of:
```
var tickContext = new VF.TickContext();
tickContext.addTickable(note).preFormat().setX(...)
```

I'll try to follow your tutorial more closely. I see that it does reference Voices:

https://github.com/0xfe/vexflow/wiki/The-VexFlow-Tutorial

In any case, would it make sense to make the clefs have a standard width? It sort of bothers me visually that the 4/4 time signatures in my two staves are offset by 1.5 pixels:

https://jsfiddle.net/Le5drt9k/6/

If the treble clef had some extra magic whitespace, then everything would all be aligned.

Anyways, thanks for your project. It's pretty awesome!

I've been working on a tool for myself to learn piano better:

https://www.pianouniversity.com/sight-reading-game/

I'm actually pretty terrible at piano so I'm trying to practice sight reading from the grand staff. I built my prototype using typescript and vexflow. It's a very V1 project, so there's lots of edge cases I don't handle, and probably a few bugs with fonts and other stuff.

Thanks again!
